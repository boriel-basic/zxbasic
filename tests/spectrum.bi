REM Check for invalid characters in a comment
REM Like this one !!!

PRINT "HELLO WORLD!"

