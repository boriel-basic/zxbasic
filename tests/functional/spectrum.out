#line 1 "spectrum.bi"



PRINT "HELLO WORLD!"

